spec/restforce_mock/schema_manager_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+describe RestforceMock::SchemaManager do
+  describe "#load_schema" do
+    it "throws an error if file is not configured" do
+      expect {
+        RestforceMock::SchemaManager.new.load_schema(nil)
+      }.to raise_error("Schema file is not defined")
+    end
+
+    it "cannot load file" do
+      expect {
+        RestforceMock::SchemaManager.new.load_schema("tmp")
+      }.to raise_error(Errno::ENOENT)
+    end
+
+    it "loads file" do
+      location = "#{File.dirname(__FILE__)}/../fixtures/schema.yml"
+      s = RestforceMock::SchemaManager.new.load_schema(location)
+      expect(s["Contact"]["Id"]).to eq({:type=>"id", :nillable=>false})
+      expect(s["Contact"]["IsDeleted"]).to eq({
+        :type=>"boolean", :nillable=>false
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.